mep3_behavior_tree/include/mep3_behavior_tree/precise_navigate_to_action.hpp

```diff
@@ -53,6 +67,17 @@ namespace mep3_behavior_tree
     getInput("goal", goal);
     getInput("behavior_tree", behavior_tree);
 
+    std::string table = config().blackboard->get<std::string>("table");
+    std::string goal_table;
+    if (table.length() > 0 && getInput("goal_" + table, goal_table)) {
+      std::cout << "Precise navigation goal for table '" \
+                << table << "' detected" << std::endl;
+      BT::pose2dFromString(goal_table, goal);
```

Comment:
```suggestion
      goal = goal_table;
```
nema potrebe  da ponovo implementirano `pose2dFromString`